src/handlers/notification-banner-handler.test.ts

```diff
@@ -0,0 +1,112 @@
+import notificationBannerHandler from "./notification-banner-handler";
+import { NextFunction, Request, Response } from "express";
+import { expect } from "chai";
+import sinon from "sinon";
+import * as parameterService from "../services/parameterStoreService";
+
+describe("Notification banner handler", () => {
+  let req: Request;
+  let res: Response;
+  let next: NextFunction;
+  let getParameterStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    req = {
+      session: {},
+      path: "/some-page",
+    } as any;
+
+    res = {
+      locals: {},
+    } as any;
+    process.env.NODE_ENV = "production";
```

Comment:
We should be a little bit careful with this - it's not the end of the world, but this will leak out of the test, and could affect other cases.

If you want to test that case, we could reset the environment after each test. Alternatively we could just skip testing the local-only behaviour - since it's not production code.